docs/assets/scss/_team.scss

```diff
@@ -12,11 +12,11 @@
   }
 
   .github-btn {
+    border: none;
```

Comment:
`border: 0` is preferred over `border: none`<br>Properties should be ordered float, width, height, margin-top, border